olm-catalog/serverless-operator/index.Dockerfile

```
@@ -0,0 +1,20 @@
+FROM quay.io/operator-framework/upstream-opm-builder
+
+RUN echo "http://nl.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories && \
+    echo "http://nl.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
+    apk update && apk add podman
+
+# Add internal registry to insecure registries.
+RUN printf "[registries.insecure]\nregistries = ['image-registry.openshift-image-registry.svc:5000']" > /etc/containers/registries.conf
+
+# Create a barebones /etc/nsswitch.conf file.
+# Required to avoid "Error: open /etc/nsswitch.conf: permission denied".
+RUN [ ! -e /etc/nsswitch.conf ] && echo 'hosts: files dns' > /etc/nsswitch.conf
+
+# Run as nobody and create a properly owned HOME directory for podman to be able to write
+# its config files. Also set that directory as a workdir to be able to create more files.
+ENV USER=nobody
+ENV HOME /home/$USER
+RUN mkdir -p $HOME && chgrp -R 0 $HOME && chmod -R g=u $HOME
```

```suggestion
RUN mkdir -p $HOME && chown -R nobody:nobody $HOME && chmod -R g=u $HOME
```

---

This doesn't actually work when run on Openshift.

---

```suggestion
RUN addgroup $USER root
RUN mkdir -p $HOME && chown -R $USER:root $HOME && chmod -R g=u $HOME
```
What about this. I've tested that on OCP. It also works everywhere, as the permissions to home dir are correct, and thous reducing confusion.

---

Done.